### DISCUSSION TITLE
Will there be syntax highlighting for HAML & Slim?

### DISCUSSION THREAD

**Discussion by satcre** [zed-industries/zed#8190] [Feature Requests]
Currently, there is no syntax highlighting for HAML and Slim templating engines. Will this be provided anytime soon?

**Comment by SomeoneToIgnore** [comment]
> anytime soon

Doubtfully, unless the community supports that.
There are open issues for both: https://github.com/zed-industries/zed/issues/4508 , https://github.com/zed-industries/zed/issues/5205 and the latter even has some activity, so might be happening, after all.
